### PR TITLE
Add Cli support for thermal control feature

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
**- What I did**
1. Allow user to show fan status
2. Allow user to show temperature information

**- How I did it**
1. Add two click command in show/main.py as the command line entry point
2. Add fanshow and tempershow to read data from database

**- How to verify it**
1. issue command "show platform fanstatus" and see the output
2. issue command "show platform temperature" and see the output

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: scripts/fanshow [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+SPEED_FIELD_NAME = 'speed'
+DIRECTION_FIELD_NAME = 'direction'
+TIMESTAMP_FIELD_NAME = 'timestamp'
+
+
+class FanShow(object):
+    def __init__(self):
+        self.db = SonicV2Connector(host="127.0.0.1")
+        self.db.connect(self.db.STATE_DB)
+
```
Comment (stephenxs) [inline scripts/fanshow]: Suggest 2 lines between functions and 3 lines between classes according to PEP8.

Comment (Junchao-Mellanox) [reply scripts/fanshow]: I am using pycharm, and it suggests 2 lines between class according to PEP8. Could you please double confirm that?

Comment (stephenxs) [reply scripts/fanshow]: yes, two lines between functions as well as classes.

File: scripts/tempershow [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+CRIT_HIGH_THRESH_FIELD_NAME = 'critical_high_threshold'
+CRIT_LOW_THRESH_FIELD_NAME = 'critical_low_threshold'
+WARNING_STATUS_FIELD_NAME = 'warning_status'
+
+
+class TemperShow(object):
+    def __init__(self):
+        self.db = SonicV2Connector(host="127.0.0.1")
+        self.db.connect(self.db.STATE_DB)
+
```
Comment (stephenxs) [inline scripts/tempershow]: still only one line between functions.  please fix it.

**Pull request merged**